docs/tutorials/gcp_gpu.md

```diff
@@ -35,6 +35,11 @@ instance section, ensure that your VM has the following properties:
     1.   In the **Operating System** option, choose **Ubuntu**.
     2.   In the **Version** option, choose **20.04 LTS**.
     3.   In the **Size** field, enter **40** (minimum).
+
+    **Alternatively, you can click the "Switch Image" button and use the image with
+    CUDA pre-installed, which lets you skip step 3. I have verified that this works
```

Comment:
Could you perhaps replace "I have verified" with "It was verified" or something like that?

Reply:
Done.